docs/logging.md

```diff
@@ -0,0 +1,58 @@
+# Logging
+
+Using Bigflow, you can easily add handler to your root logger that will send all loggers calls to the Cloud Logging.
+In the CL, you can filter logs per project id/logger name and workflow id. 
+
+## Getting started
+To use logging, you have to install `bigflow` with `logger` module - `bigflow[log]`.
+To create logger with the CL handler, you have to use `configure_logging` method.
```

Comment:
To create logger with the CL handler, you have to use the `configure_logging` method.